s4e-web/src/app/state/profile/profile.query.ts

```diff
@@ -7,11 +7,28 @@ import {map} from 'rxjs/operators';
 
 @Injectable({providedIn: 'root'})
 export class ProfileQuery extends Query<ProfileState> {
+  public isAdmin = false;
+  public isManager = false;
 
   constructor(protected store: ProfileStore) {
     super(store);
   }
 
+  public loadRole() {
+    this.isAdmin$().subscribe(isAdmin => this.isAdmin = isAdmin);
+    this.selectCanSeeInstitutions().subscribe(isManager => this.isManager = isManager);
+  }
+
+  public clearRole() {
+    this.isAdmin = false;
+    this.isManager = false;
+  }
```

Comment:
You can use store for this. And only provide methods like `isManager()` and `isAdmin()` instead of public fields.
Storing state in the query is not the best idea.